File: examples/golang/mypackage/handler.go
package main

import (
	"encoding/json"

	"github.com/scaleway/scaleway-functions-go/events"
	"github.com/scaleway/scaleway-functions-go/lambda"
)

// Handler - Handle event
func Handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
	response := map[string]interface{}{
		"message": "My Custom Package",
	}

	responseB, err := json.Marshal(response)
	if err != nil {
		return events.APIGatewayProxyResponse{}, err
	}

	return events.APIGatewayProxyResponse{
		Body:       string(responseB),
		StatusCode: 200,
	}, nil
}

func main() {
	lambda.Start(Handler)
}

File: examples/php8/handler.php
module.exports.handle = (event, context, callback) => {
  const result = {
    message: 'Hello from Serverless Framework and Scaleway Functions :D',
  };
  const response = {
    statusCode: 200,
    headers: {"Content-Type": ["application/json"]},
    body: JSON.stringify(result),
  };

  // either return cb(undefined, response) or return response
  return response;
};
